packages/intrepid2/unit-test/Projection/test_convergence_HEX.hpp

```diff
@@ -0,0 +1,1154 @@
+// @HEADER
```

Comment:
I agree. It would make the code a bit harder to read but it would reduce the redundancy a lot.
Would you like me to do this before merging this?

Reply:
It's really up to you.  If you can do it quickly and easily, then please do.  If it might take a few days to get to, I think it's probably good to finish this PR, and then update with the refactor.